nautobot/docs/development/apps/index.md

```diff
@@ -25,28 +22,13 @@ The Nautobot App architecture allows for Apps to do any or all of the following:
 * **Add home page content.** An App can add custom items or custom panels to the Nautobot home page.
 * **Add content to existing model detail views.** An App can inject custom HTML content within the view of a core Nautobot model. This content can appear in the left column, right column, or full width of the page, and can also include custom buttons at the top of the page.
 
-+++ 1.2.0
-    * **Add a banner.** An App can add a custom banner to the top of any appropriate views.
-
-+++ 1.4.0
-    * **Add extra tabs to existing model detail views.** An App can inject additional tabs which will appear at the end of the object detail tabs list.
 
 ### Extend and customize existing Nautobot functionality
 
 * **Add custom validation logic to existing data models.** An App can provide additional logic to customize the rules for validating created/updated data records.
 * **Provide Jobs.** An App can serve as a convenient way to package and install [Jobs](../../user-guide/platform-functionality/jobs/index.md).
 * **Add additional Git data types.** An App can add support for processing additional types of data stored in a [Git repository](../../user-guide/platform-functionality/gitrepository.md).
 
-+++ 1.1.0
-    * **Register additional Jinja2 filters.** An App can define custom Jinja2 filters to be used in computed fields, webhooks, custom links, and export templates.
-
-+++ 1.2.0
-    * **Populate extensibility features in the database.** An App can add content to the Nautobot database when installed, such as automatically creating new custom fields, relationships, and so forth.
-
-    * **Add additional secrets providers.** An App can add support for retrieving [Secret](../../user-guide/platform-functionality/secret.md) values from additional sources or external systems.
-
-+++ 1.4.0
-    * **Override already-defined views.** An App can define a view which can be set to override a view from the core set of views or another App's view.
```

Comment:
These items should be rolled into the above bullet list rather than removed entirely.

Reply:
Sounds good, I am adding these  all under the same bullet heading `### Extend and customize existing Nautobot functionality` without the tags